apps/custom-ui-app-docusign/server/util/helper.js

```diff
@@ -0,0 +1,145 @@
+const axios = require("axios");
+const debug = require('debug')('app:helper');
+const querystring = require('querystring');
+
+// Generates a new token based on the refresh token
+const getNewToken = async (refresh_token) => {
+    try {
+        return axios({
+            method: "POST",
+            url: "https://oauth.pipedrive.com/oauth/token",
+            headers: {
+                Authorization: `Basic ${Buffer.from(process.env.CLIENT_ID + ":" + process.env.CLIENT_SECRET).toString("base64")}`,
+                'content-type': 'application/x-www-form-urlencoded'
+            },
+            data: querystring.stringify({
+                grant_type: "refresh_token",
+                refresh_token,
+            }),
+        });
+    } catch (error) {
+        debug(error);
+        throw new Error("Getting new token from refresh token failed");
+    }
+};
+
+// Gets the page context for rendering the HBS templates
+function getPageContext(company_id, item_id, details) {
```

Comment:
Why is this function needed?

Reply:
Good point. Seems like I wrote it for a different reason but its not required.